R/loo.R

```diff
@@ -256,20 +258,22 @@ loo.function <-
       r_eff <- prepare_psis_r_eff(r_eff, len = N)
     }
 
-    psis_list <- parallel_psis_list(N = N,
+    psis_list <- parallel_importance_sampling_list(
+                                    N = N,
                                     .loo_i = .loo_i,
                                     .llfun = .llfun,
                                     data = data,
                                     draws = draws,
                                     r_eff = r_eff,
                                     save_psis = save_psis,
                                     cores = cores,
-                                    ...)
+                                    ...,
+                                    is_method = is_method)
```

Comment:
is there a reason `is_method` is coming after `...` in the function call?

Reply:
No. This could be fixed. You think ... always should be in the end?

Reply:
It does not need to be at the end but there should be clear reasons for not having it at the end, which i don't see in this case.